### PR TITLE
Some links in the threat hunting dashboard table do not redirect or have no links

### LINKED ISSUE #6830: Some links in the Threat Hunting dashboard table do not redirect or have no links
**Description**

The values of the following fields in the Threat Hunting dashboard table do not redirect or do not have links:

`rule.mitre.id` does not redirect
`rule.mitre_techniques` no link to redirect to

https://github.com/user-attachments/assets/18a07267-0ef1-4fbb-97fb-e10cd41061a1

### PR DESCRIPTION
**Pull request by chantal-kelm**
### Description

This PR fix the issue some links in the threat hunting dashboard table do not redirect or have no links
 
### Issues Resolved

#6830 

### Evidence

https://github.com/user-attachments/assets/0b49c93b-7361-4f32-bd7d-67df8042c1d7

### Test

- Go to the Threat Hunting tab, in the dashboard table click on one of the values in the `rule.mitre.id` field and check that it redirects correctly.

- Go to the Threat Hunting tab, in the dashboard table click on one of the values in the `rule.mitre_techniques` field and check that it redirects correctly.

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by jbiset — APPROVED** [review]
## CR: :green_circle: 

## Test UI: :green_circle: 

### Evidence

[Evidence.webm](https://github.com/user-attachments/assets/bbb8911d-99f9-4d96-a061-892f2af3c3b2)

File: plugins/main/public/components/common/wazuh-discover/render-columns.tsx [hunk 1]
```diff
@@ -3,26 +3,23 @@ import { EuiLink } from '@elastic/eui';
...
-    }
+    onClick={e => {
+      e.preventDefault();
+      NavigationService.getInstance().navigateToApp(mitreAttack.id, {
+        path: `#/overview?tab=mitre&tabView=intelligence&tabRedirect=techniques&idToRedirect=${technique}`,
+      });
+    }}
   >
     {technique}
   </EuiLink>
```
Comment (yenienserrano) [inline plugins/main/public/components/common/wazuh-discover/render-columns.tsx:25]: Perhaps you could use the RedirectAppLinks to get the reference to where you are navigating to. As it is in line 35.

```
  <RedirectAppLinks application={getCore().application}>
    <EuiLink
      href={NavigationService.getInstance().getUrlForApp(mitreAttack.id, {
        path: `#/overview?tab=mitre&tabView=intelligence&tabRedirect=techniques&idToRedirect=${technique}`,
      })}
    >
      {technique}
    </EuiLink>
  </RedirectAppLinks>
```

image

Comment (chantal-kelm) [reply plugins/main/public/components/common/wazuh-discover/render-columns.tsx]: Fix in commit https://github.com/wazuh/wazuh-dashboard-plugins/pull/6845#discussion_r1684130631

**Review by yenienserrano — DISMISSED** [review]
CR: 🟢 
Test: 🟢 

image

**Review by jbiset — DISMISSED** [review]
## CR new changes: :green_circle: 

## Test UI changes: :green_circle: 

### Evidence

image

image

**Review by jbiset — APPROVED** [review]

**Review by yenienserrano — APPROVED** [review]

**Comment by Desvelao** [comment]
This pull request added the field `rule.mitre_techniques` to the threat hunting table and it should be done.

- https://github.com/wazuh/wazuh-dashboard-plugins/pull/6845/files#diff-cdece0cf0b472e569103001596b378fe53b368615c9ff11c3fb805409d6e2e31R52-R54
- https://github.com/wazuh/wazuh-dashboard-plugins/pull/6845/files#diff-cdece0cf0b472e569103001596b378fe53b368615c9ff11c3fb805409d6e2e31R25-R27

We will create a pull request to remove this field.

**Pull request merged**